tests/ValueParsers/IsoTimestampParserTest.php

```diff
@@ -151,18 +157,29 @@ public function validInputProvider() {
 			'-0000000000000001-01-01T00:00:00Z (Gregorian)' => array(
 				'-0000000000000001-01-01T00:00:00Z',
 				TimeValue::PRECISION_DAY,
+				$gregorian
 			),
 			'-00000000001-01-01T00:00:00Z (Gregorian)' => array(
 				'-0000000000000001-01-01T00:00:00Z',
 				TimeValue::PRECISION_DAY,
+				$gregorian,
+				$julianOpts // overridden by explicit calendar in input string
+			),
+			'-00000000001-01-01T00:00:00Z (Julian)' => array(
+				'-0000000000000001-01-01T00:00:00Z',
+				TimeValue::PRECISION_DAY,
+				$julian,
+				$gregorianOpts // overridden by explicit calendar in input string
```

Comment:
Important tests. I just checked if it was like this before (the option does have lower priority than the string) and yes, it was.